docs/Project-Configuration/Project-Commands.md

```diff
@@ -540,7 +540,14 @@ permissions:
 Certain instances require more time for SSH access to become available.
 If the user plans to execute commands on the remote host, then waiting
 for SSH access to become available is mandatory. Below is an Evergreen
-function that probes for SSH connectivity:
+function that probes for SSH connectivity.
+
+Note, however, an important bash caveat! By default Evergreen implements
```

Comment:
I feel like someone reading this without reading the entire original thread + having some pre-existing bash knowledge (+ knowing that the default shell for `shell.exec` is _not_ bash) won't understand what they're supposed to take away from this warning. It feels like this either needs its own YAML example separate from the one described above or apply the real fix to the `script` example below and explain why there.

Reply:
I edited the example. I also edited some of the "bash" to read "shell".